continuous-delivery/pack.sh

```diff
@@ -39,7 +39,8 @@ mkdir $UNZIP
 tar -xf aws-crt-$CURRENT_TAG.tgz -C $UNZIP
 PACK_FILE_SIZE_KB=$(du -sk $UNZIP | awk '{print $1}')
 echo "Current package size: ${PACK_FILE_SIZE_KB}"
-if expr $PACK_FILE_SIZE_KB \> "$((18000))" ; then
+# TODO: Revert before merge. This is temporary for testing
+if expr $PACK_FILE_SIZE_KB \> "$((40000))" ; then
```

Comment:
The package size will increase after the addition of the Musl binary, but I don't know the exact number yet. The test pipeline resulted in a size of over 30,000KB, which seems incorrect. Maybe I will let the release pipeline fail and then adjust the size accordingly.